File: UBB_CS_II_sem_2/SDI/Laboratory_2/src/main/java/TrainStations/Model/Train.java
package TrainStations.Model;

public class Train implements DTOCustomObj{
    private final int id;
    private final String type;
    private final String color;
    private final String fabricationDate;

    public String getType() {
        return type;
    }

    public String getColor() {
        return color;
    }

    public String getFabricationDate() {
        return fabricationDate;
    }

    public Train(int id, String type, String color, String fabricationDate){
        this.id = id;
        this.type = type;
        this.color = color;
        this.fabricationDate = fabricationDate;
    }

    @Override
    public String toString() {
        return "Train{" +
                "type='" + type + '\'' +
                ", color='" + color + '\'' +
                ", fabricationDate='" + fabricationDate + '\'' +
                '}';
    }
}

File: UBB_CS_II_sem_2/DBMS/Example2/DBMS_CRUD/Model/Train.cs
using DBMS_CRUD.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace DBMS_CRUD.NewFolder
{
    public class Train: DBObject
    {
        private readonly int id;
        private readonly string type;
        private readonly string color;
        private readonly DateTime dateTime;

        public Train( string type, string color, DateTime dateTime)
        {
            
            this.type = type;
            this.color = color;
            this.dateTime = dateTime;
        }

        public string Color => color;

        public string Type => type;

        public DateTime DateTime => dateTime;

        public int getId() => id;

        public override string ToString()
        {
            String returnString = "";
            returnString += id + " " + type + " " + color + " " + dateTime.ToString();
            return returnString;
        }
    }
   
}
